tests/runtimes/android/sbg_tests.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+"""
+Test for android runtime static binding generator
+"""
+# pylint: disable=invalid-name
+
+import os
+
+from core.base_test.tns_test import TnsTest
+from core.utils.device.device_manager import DeviceManager
+from core.utils.file_utils import File, Folder
+from core.settings.Settings import Emulators, Android, TEST_RUN_HOME, AppName
+from data.templates import Template
+from products.nativescript.tns import Tns
+
+APP_NAME = AppName.DEFAULT
+
+
+class SBGTests(TnsTest):
+
+    @classmethod
+    def setUpClass(cls):
+        TnsTest.setUpClass()
+        cls.emulator = DeviceManager.Emulator.ensure_available(Emulators.DEFAULT)
+        Folder.clean(os.path.join(TEST_RUN_HOME, APP_NAME))
+
+    def test_300_fail_build_when_sbg_bindings_file_is_missing(self):
+        Tns.create(app_name=APP_NAME, template=Template.HELLO_WORLD_NG.local_package, update=True)
```

Comment:
Do we really need NG app, can we use Template.MIN_JS, this will make test much faster.

Reply:
Yes, it can be JS as well. Changed.